release-notes/opensearch-anomaly-detection.release-notes-1.0.0.0-beta1.md

```diff
@@ -4,9 +4,6 @@ Compatible with OpenSearch 1.0.0
 
 ### Infrastructure
 
-* Fix: use AWS_SECRET_ACCESS_KEY. ([#3](https://github.com/opensearch-project/anomaly-detection/pull/3))
-* Delete ad-cli ([#5](https://github.com/opensearch-project/anomaly-detection/pull/5))
-* Fix access key typo, added VSCode gitignores ([#8](https://github.com/opensearch-project/anomaly-detection/pull/8))
 * Build using local dependencies, re-enable CI ([#13](https://github.com/opensearch-project/anomaly-detection/pull/13))
```

Comment:
I'd remove these too, the only news here is that plugin works with OpenSearch 1.0.

Reply:
Are you suggesting removing the remaining two PRs listed under `Infrastructure`?

Reply:
Yes, because in the previous version none of this stuff was broken and doesn't represent an improvement or a change.

Reply:
Makes sense. I've removed